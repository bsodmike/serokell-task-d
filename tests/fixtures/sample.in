10
add "buy bread" #groceries
add "buy milk" #groceries
add "call parents" #relatives

search buy
search a