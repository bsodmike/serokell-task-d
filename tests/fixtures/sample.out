0
1
2

# search #groceries
2 item(s) found
1 "buy milk" #groceries
0 "buy bread" #groceries

# search buy
2 item(s) found
1 "buy milk" #groceries
0 "buy bread" #groceries

# search a
2 item(s) found
2 "call parents" #relatives
0 "buy bread" #groceries

# done 0
done
1 item(s) found
2 "call parents" #relatives

# done 2
done
0 item(s) found
